exchanges/binance/binance_test.go

```diff
@@ -2335,7 +2335,8 @@ func TestGetHistoricCandlesExtended(t *testing.T) {
 }
 
 func TestBinance_FormatExchangeKlineInterval(t *testing.T) {
```

Comment:
🗻 OOS But DriveBy if you want
```suggestion
func TestFormatExchangeKlineInterval(t *testing.T) {
```